src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-box" data-qa="card">
+      <img
+        class="product-box__item"
+        src="/images/imac.jpeg"
+        alt="apple-monoblock"
+      >
+      <h2 class="product-box__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <p class="product-box__id-code">Код товара: 195434</p>
+
+      <div class="five-stars product-box__five-stars">
+        <ul class="five-stars__list five-stars__list--4">
```

Comment:
```suggestion
        <ul class="stars__list five-stars__list--4">
```
Думаю можно просто stars назвать блок, зачем количество звезд выносить в название?